starter_code/hangman.js

```diff
@@ -1,36 +1,51 @@
 var hangman;
 
-// function Hangman() {
 
-// }
-
-// Hangman.prototype.getWord = function () {
-
-// };
-
-// Hangman.prototype.checkIfLetter = function (keyCode) {
-
-// };
-
-// Hangman.prototype.checkClickedLetters = function (key) {
+function Hangman() {
+  this.words = [];
+  this.secretWord = '';
+  this.letters = [];
+  this.guessedLetter = '';
+  this.errorsLeft = 10;
+}
+
+Hangman.prototype.getWord = function () {
+  this.secretWord = toString(this.words[Math.floor(Math.random*this.words.length)]);
+  return this.secretWord;
+};
 
-// };
+Hangman.prototype.checkIfLetter = function (keyCode) {
+  if (keyCode <= 43) {
```

Comment:
Care with the code. You should look if they if is `keyCode > 64 && keyCode < 91`
Here a page where you have a table:
https://www.cambiaresearch.com/articles/15/javascript-char-codes-key-codes